features/visitor_can_create_new_task.feature

```diff
@@ -0,0 +1,27 @@
+Feature: A visitor can create new task
+    As a visitor,
+    In order to get my furniture refurbished
+    I would to be able create a new task
+
+    Scenario: Create task [Happy path]
+        When I am at the index page
+        And I click on 'New Task' link
+        Then I should be on New Task page
+        And I fill in 'Title' with 'My Task'
+        And I fill in 'Description' with 'Really old table that needs refurnishing'
+        And I fill in 'Budget' with '100'
+        And I fill in 'Location' with 'Stockholm'
+        And I click on 'Post Task'
+        Then there should be a 'New Task' in the database
+        And I should see 'Task successfully created'
+
+    Scenario: Create Task [Sad Path]
+        When I am at the index page
+        And I click on 'New Task' link
+        Then I should be on New Task page
```

Comment:
No need to have this assertion in the middle of the scenario. Not a blocker though.